test/index.js

```diff
@@ -114,6 +114,12 @@ describe('node-precinct', function() {
     var result = precinct(read('typescriptWithError.ts'));
     assert(result.length === 0);
   });
+  
+  it('grabs dependencies of typescript tsx files', function() {
```

Comment:
When you remove the on by default for that tsx option, consider setting that option in the test.